build.gradle.kts

```diff
@@ -7,6 +7,8 @@ plugins {
     id("com.github.johnrengelman.shadow") version "5.1.0"
     id("org.jetbrains.dokka") version "0.10.1"
     id("org.jlleitschuh.gradle.ktlint") version "9.4.1"
+//  https://github.com/xvik/gradle-use-python-plugin
```

Comment:
This doesn't do what I had intended, unfortunately. The PSI resolver still does not recognize any builtins in tests :^(

Reply:
Okay, let's leave it to the next PR